src/codeGenerators/orchestration/files/toOrchestration.ts

```diff
@@ -392,6 +393,16 @@ const prepareSetupScript = (file: localFile, node: any) => {
   file.file = file.file.replace(/CONSTRUCTOR_CALL/g, constructorCall);
 }
 
+const prepareStartupScript = (file: localFile, node: any) => {
+  let constructorCall = ``;
+  if (!node.functionNames.includes('cnstrctr')) {
+    return;
+  } else {
+    
```

Comment:
This function doesn't seem to do anything other than line 397. Can it be removed and replaced with just let constructorCall = ``;?